docs/docs.json

```
@@ -3,11 +3,11 @@
   "theme": "palm",
   "name": "AgentKit by Inngest",
   "colors": {
-    "primary": "#189c70",
-    "light": "#2fe4a8",
-    "dark": "#2fe4a8"
+    "primary": "#2C9B63",
+    "light": "#9ADAB3",
+    "dark": "#016239"
   },
-  "favicon": "/brand/favicon-june-2024-dark.png",
+  "favicon": "/brand/favicon.svg",
```

```suggestion
  "favicon": "/brand/favicon.png",
```
Can we use a png for now? svg usage in a favicon isn't as well supported in Safari still: https://caniuse.com/link-icon-svg